packages/ui/src/components/Notification/index.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Theme } from '@emotion/react'
+import { ClassNames, Global, css, useTheme } from '@emotion/react'
+import type { ReactNode } from 'react'
+import {
+  ToastContainer as BaseToastContainer,
+  Slide,
+  toast as baseToast,
+} from 'react-toastify'
+import type {
+  Theme as ThemeToastify,
+  ToastOptions,
+  TypeOptions,
+} from 'react-toastify'
+import style from 'react-toastify/dist/ReactToastify.min.css'
+import { Button } from '../Button'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+const PREFIX = '.Toastify'
+type StylesProps = {
+  theme: Theme
+}
+
+type CloseButtonProps = {
+  closeToast: (e: React.MouseEvent<HTMLElement>) => void
```

Comment:
```suggestion
  closeToast: (event: React.MouseEvent<HTMLElement>) => void
```

We tend to avoid one letter to define a variable and make it as much explicit as possible